src/near-rpc-provider.ts

```diff
@@ -372,4 +374,23 @@ export class NearRpcProvider extends JsonRpcProvider {
       })
     }
   }
+
+  async getAccessKey(addressOrName: string, publicKey: string, blockTag: BlockTag): Promise<NearAccessKeyResponse> {
+    let getaccessKeyParams: GetAccessKeyParams = {
```

Comment:
```suggestion
    let getAccessKeyParams: GetAccessKeyParams = {
```

make sure to also change every other occurance